### ISSUE TITLE
Magnitude limit changeable with keyboard shortcuts

### ISSUE THREAD

**Issue by cumuluscumulus** [Stellarium/stellarium#174]
Hi Stellarium,

it would be very helpful, if the magnitude limit of the displayed stars can be changed up and down with keyboard shortcuts, stepwidth for example 0,2 mag 

Thank you!

**Comment by gzotti** [comment]
I don't like this idea.

This limit is computed automatically from sky conditions to give a pretty good idea for naked-eye conditions. You can artificially influence "absolute" and "relative" scale for stars and since today there are keyboard shortcuts for Bortle light pollution index.

**Comment by cumuluscumulus** [comment]
Your argument is ok for "naked-eye conditions". 
But only for this! 
If not, why can Stellarium display stars and objects with less than 6 mag?

**Comment by cumuluscumulus** [comment]
@ alex-w:
when I want to "analyze" which stars / objects are on my images.

@ gzotti:
ok. But where is the argument against my request? 
There are a lot of things easy to change: grids, constellation lines, meteor showers etc. Why not a magnitude limit? Only because you "don't like it?"

**Comment by gzotti** [comment]
There are already two switches which do what you want: relative and absolute scale.

**Comment by gzotti** [comment]
@alex-w  No, OP wants to use a magnitude switch which does the same that currently the star scales switches do, but in a way which would use the mag scale instead of unit-less factors. This would at least require a re-development of the way star visibility is computed (the RCmag computation), and possibly some interfacing with the sky brightness code to maybe switch on stars at sunset or so.  OP is free to submit a PR which shall allow at least the same possibilities and auto-estimate computation as the current system. The next user will request that visual acuity will be taken into account. Add another switch. The next will start with yellow-filtering by aging eyes. Add another switch. And so on. These two existing parameters can be set to do what OP wants to do, analyze what stars are on his images. 

Grids and meteor showers are easy to change because some developers have made it easy. This did not grow out of the clouds. Some users want to influence grid-spacing manually. For the time being we opted for less switches because we have a working system already which fits most users, and want to concentrate on real improvements.

**Comment by cumuluscumulus** [comment]
"@gzotti I think OP meant changing values for option “Limit magnitude” for stars."
Thats what I mean.  
Two shortcuts instead using  F4 and arrows up or down with a disturbing large menue over the stars.
Try it.

**Comment by gzotti** [comment]
Try it yourself then. 
Or use a webbrowser on a second computer/tablet/smartphone with RemoteControl plugin if you want to avoid the main GUI.

**Comment by cumuluscumulus** [comment]
Thanke you!

**Comment by alex-w** [comment]
@cumuluscumulus please check version 0.90.0.16018 (0.18.0.9)
